app/uk/gov/hmrc/gform/views/hardcoded/pages/save_acknowledgement.scala.html

```diff
@@ -21,15 +21,20 @@
 @import uk.gov.hmrc.gform.sharedmodel.formtemplate.{FormTemplateId, _}
 @import uk.gov.hmrc.gform.config.FrontendAppConfig
 @import uk.gov.hmrc.gform.views.hardcoded.pages._
+@import java.time.format.DateTimeFormatter
+
 
 @(
+    envelopeExpiryDate: Option[EnvelopeExpiryDate],
     formTemplate: FormTemplate,
     call: Call,
     lang: Option[String],
     frontendAppConfig: FrontendAppConfig
 )(implicit messages: Messages)
 
+@formatter = @{DateTimeFormatter.ofPattern("dd MMMM yyyy")}
 @heading = @{"Your " + formCategory(formTemplate) + " has been saved"}
+@expiryDate = @{envelopeExpiryDate.fold("for 28 days")("until " + _.ldt.format(formatter))}
```

Comment:
No!

The fold empty case is only for existing forms without an expiry date present.  They have been created with 30 days, so leave the text saying 30

Reply:
Apologies I completely forgot about this situation when i changed the expiry date in the config file.